databaseFunctions.js

```diff
@@ -0,0 +1,72 @@
+const { db, end } = require("./database.js");
+
+// Takes in task name for creation and outputs the id of the created task
+async function addTask(taskName) {
+  try {
+    const query = `INSERT INTO task (name, completed, created_date)
+                        VALUES ($1, false, NOW())
+                        RETURNING id;`;
+
+    const res = await db.one(query, [taskName]);
+    return res;
+  } catch (err) {
+    console.error("Error adding in task: " + err);
```

Comment:
Since this is visible to the user, we probably want a different message here. The doesn't make a lot of sense 🤔

Reply:
Changed error handling so error messages only display once, so no longer applicable.